llvm/lib/Transforms/PeepholeOptimizationCourse/Func/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+add_llvm_component_library(
+    LLVMFunc
```

Comment:
I see that you link `LLVMPasses` against `LLVMFunc`, but you use it in your passes, which are not linked against it. How is that working?

Reply:
Could you please elaborate on what you mean by linking `LLVMPasses` against `LLVMFunc`, because the code you highlighted just adds Func as a module to LLVM

Reply:
I mean you added Func [here](https://github.com/ggladilov/llvm-project/pull/13/files#diff-94eed4ae5a8236951da49d8a51cdd88ee9c9f557f290a3048f0ab11c227bf381R32), but I don't see anything similar for passes libraries. How does that work?

Reply:
Right, you confused me here because there is `ReplaceInt8AddNewPassManager` in that CMakeLists file, because that's how we do it using the new pass manager. My assumption would be that we don't actually need `Func` there.

Reply:
I just tried removing it and it still works, so I suppose I was correct